trident/light_ray.py

```diff
@@ -517,13 +529,14 @@ def make_light_ray(self, seed=None, periodic=True,
         self._data = {}
         # temperature field is automatically added to fields
         if fields is None: fields = []
-        if ('gas', 'temperature') not in fields:
-           fields.append(('gas', 'temperature'))
+        if (('gas', 'temperature') not in fields) and \
+           ('temperature' not in fields):
+            fields.append(('gas', 'temperature'))
         data_fields = fields[:]
         all_fields = fields[:]
         all_fields.extend(['l', 'dl', 'redshift'])
-        all_fields.extend(['x', 'y', 'z'])
-        data_fields.extend(['x', 'y', 'z'])
+        all_fields.extend([('gas','x'),('gas','y'),('gas','z')])
+        data_fields.extend([('gas','x'),('gas','y'),('gas','z')])
         if use_peculiar_velocity:
```

Comment:
This change wasn't actually meant to be part of the PR, was just a random bug fix. I don't remember exactly the problem here, but if there is a more elegant solution to this I'm happy to change it